jina/resources/executors._merge_all.yml

```diff
@@ -5,6 +5,8 @@ metas:
 requests:
   on:
     [SearchRequest, TrainRequest, IndexRequest]:
-      - !ReduceAllDriver {}
+      - !ReduceAllDriver
+        with:
+          traversal_paths: ['m']
```

Comment:
So for me ideally I'd like to minimize the number of times we have exceptions / configuration changes at the yaml file. These should be available to be made in module/examples etc, but not be part of the core resources, I'd say. These should be as easy to read as possible, with changes pushed towards the code itself

Reply:
the point is flexibility, if we want to merge `matches` it will be `matches` (but sometimes one may want to merge other things). This is some work we need to make to ease things to the user, but at this point, we need to make a decision

Reply:
let me do something here

Reply:
So what is the *sensible* default here? I'd say matches.